### PR TITLE
SONARUNNER-143 New API to retrieve issues produced by the analysis

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

**Comment by dbmeneses** [comment]
I will still add ITs before merging

File: sonar-runner-api/src/main/java/org/sonar/runner/api/Issue.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  private final String componentKey;
+  private final Integer line;
+  private final String message;
+  private final String rule;
+  private final String status;
+  private final String resolution;
+  private final boolean isNew;
+  private final String assignee;
+  private final Double effortToFix;
+  private final Date creationDate;
```
Comment (henryju) [inline sonar-runner-api/src/main/java/org/sonar/runner/api/Issue.java]: I'm wondering if we need to expose effortToFix and creationDate. This is not used currently on Eclipse side.

**Comment by henryju** [comment]
LGTM

File: sonar-runner-api/src/main/java/org/sonar/runner/api/Issue.java (hunk 1 above)
Comment (dbmeneses) [inline sonar-runner-api/src/main/java/org/sonar/runner/api/Issue.java]: Ok, I'll remove it. We can always add fields later.

**Pull request closed without merging**